commons/types/rate_limit.go

```diff
@@ -62,6 +62,13 @@ func (this *RateLimit) Set(window Window, txn *badger.Txn, cache *cache.Cache) e
 
 func (this *RateLimit) cache(window Window, cache *cache.Cache) {
 	cache.Set(getAccountRateLimitKey(this.Address), this.Existing, TransactionCacheTTL)
+	utils.Debug("Current Window TTL = ", window.TTL)
```

Comment:
@restevens402 Do we have anything recording cache misses? It might be helpful to see that information over time.

Reply:
Well, the ratio of hits and misses.

Reply:
No, we do not at this time